### ISSUE TITLE
Stream API example seems broken.

### ISSUE THREAD

**Issue by xchiptune** [oobabooga/text-generation-webui#417]
Running the API example for the stream mode I get the following error:

`FileNotFoundError: [Errno 2] No such file or directory: 'softprompts/What I would like to say is the following: .zip'`

The non-stream mode one works fine. Think maybe a recent update changed the form format? There's no mention of it even being possible to upload a softprompt in the API example.

I'm running the webui with `python server.py --load-in-4bit --model llama-13b --listen`

**Comment by oobabooga** [comment]
I can confirm the issue.

**Comment by thot-experiment** [comment]
Same bug, anyone know which commit this broke on? Was working maybe 2 or 3 days ago.

**Comment by thot-experiment** [comment]
So it looks like this API is something that's autogenerated by Gradio itself (sorry again for the naivete here I really have no idea what's going on) and because of the recent changes in the UI the shape of this API has changed as well. (I think we should try to figure out how to avoid this in the future)

For the time being here's an example of a working payload so anyone blocked by this can hack together working code for the time being and perhaps to give people an idea where to look next time this breaks.

Everything behaves the same as the example with the exception of the two lines with comments.
```js
{
  "fn_index": 9, //this was '7' originally and now must be '9'
  "data": [
    "Common sense questions and answers\\n\\nQuestion: What is the atomic number of lithium?\\nFactual answer:",
    200,
    true,
    1.99,
    0.18,
    1,
    1.15,
    1, //this parameter is new and represents the "encoder_repetition_penalty"
    30,
    0,
    0,
    1,
    0,
    1,
    false
  ],
  "session_hash": "f420f69f"
}
```

P.S. Really seems stupid to add a parameter in the middle of an array of nameless parameters...

**Comment by LTSarc** [comment]
@thot-experiment the parameters *are* named in api-example-stream.py

I am going to assume that simply hacking the FN index to be 9 will fix it, given that otherwise the numbers match up with that working payload.

**Comment by oobabooga** [comment]
Also, if you are interested in APIs, make sure to also check https://github.com/oobabooga/text-generation-webui/pull/342

It is possible to create custom APIs using extensions.

**Comment by cpwilson1989** [comment]
This appears to have raised its head again, but with the corrent fn_index now being 11!

**Comment by Anonym0us33** [comment]
```
\modules\chat.py", line 437, in load_character
    file_contents = open(filepath, 'r', encoding='utf-8').read()
FileNotFoundError: [Errno 2] No such file or directory: 'characters\\instruction-following\\TavernAI-Fisherwoman Minori.json'
```
not sure if this bug is continuing or if this is new but i don't want to waste time with a new issue
```
>dir /d
Example.png                      
Example.yaml                       TavernAI-Emma.json                
[instruction-following]            TavernAI-Emma.png
```
```
\instruction-following>dir /d
 Alpaca.yaml           Vicuna.yaml
[..]                  Open Assistant.yaml
```
